db/migrate/20231128103530_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :content
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
@gemmen29 Hi! i hope you would be fine could you please explain further what you actually want me to do because I didn't get it.. as far I know that  when creating posts table if we specify "rails g model post title:string text:text comments_counter:string likes_counter:string user:references"  command ... it automatically associates the foreign key to users as author_id ...

Reply:
Hi @zunairkhan811 No, the foreign key now is `user_id`. If you look at the generated `schema.rb` file you will find that the generated foreign key is `user_id` instead of `author_id`.
![image](https://github.com/zunairkhan811/blog-app/assets/24703171/dd3aa9ca-88ff-481c-b438-f294932903ca)

Reply:
@gemmen29 Thanks for elaborating bro. The issue has been resolved. 😃